mozbench/static/webaudio-benchmark/benchmarks.js

```diff
@@ -229,6 +259,62 @@ registerTestCase({
   name: "Granular synthesis"
 });
 
+registerTestCase({
+  func: function() {
+    var samplerate = 44100;
+    var duration  = 30;
+    var oac = new OfflineAudioContext(1, duration * samplerate, 44100);
+    var offset = 0;
+    while (offset < duration) {
+      var note = oac.createOscillator();
+      var env = oac.createGain();
```

Comment:
You call this "gain" above and "enveloppe" below. Unless there's a reason for the different terminology in the different test cases, I think it would be best if you used a consistent name.